codeception.yml

```diff
@@ -0,0 +1,17 @@
+actor: Tester
+paths:
+    tests: tests
+    log: tests/_output
```

Comment:
Why the underscore?  Is this convention I'm unaware of?

You've added the `_output` folder but not `_data`